EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py

```diff
@@ -81,18 +81,19 @@
 
 
 # ---------- pixels ----------
-ctppsPixelDAQMappingESSourceXML = cms.ESSource("CTPPSPixelDAQMappingESSourceXML",
-  verbosity = cms.untracked.uint32(0),
-  subSystem= cms.untracked.string("RPix"),
-  configuration = cms.VPSet(
-  # example configuration block:
-  cms.PSet(
-    validityRange = cms.EventRange("1:min - 999999999:max"),
-    mappingFileNames = cms.vstring("CondFormats/CTPPSReadoutObjects/xml/rpix_mapping_220_far.xml"),
-    maskFileNames = cms.vstring("CondFormats/CTPPSReadoutObjects/xml/rpix_channel_mask_220_far.xml")
-    )
-  )
-)
+## this part needed only for direct access to xmls. No DB use.
+#ctppsPixelDAQMappingESSourceXML = cms.ESSource("CTPPSPixelDAQMappingESSourceXML",
+#  verbosity = cms.untracked.uint32(0),
+#  subSystem= cms.untracked.string("RPix"),
+#  configuration = cms.VPSet(
```

Comment:
if a label needs to be, please leave it configurable,
why was this part of the python removed ?